.travis/amd_sdk.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Original script from https://github.com/gregvw/amd_sdk/
```

Comment:
@SteveBronder, is this script going to be stable? It looks like the README says it isn't. 
> This script no longer works as AMD has changed their website to add another page where you must accept terms and conditions to download.

Reply:
The original script does fail, but this version is updated to get past the terms and conditions